src/editor/Sidebar.js

```diff
@@ -156,13 +161,35 @@ const Sidebar = ( {
 				</>
 			) }
 			{ placement === 'inline' && (
-				<RangeControl
-					label={ __( 'Approximate Position (in percent)', 'newspack-popups' ) }
-					value={ trigger_scroll_progress }
-					onChange={ value => onMetaFieldChange( 'trigger_scroll_progress', value ) }
-					min={ 0 }
-					max={ 100 }
-				/>
+				<>
+					<SelectControl
+						label={ __( 'Trigger', 'newspack-popups' ) }
+						help={ __( 'The event to trigger the prompt.', 'newspack-popups' ) }
```

Comment:
This is not about an event, just an insertion point. 

```suggestion
						help={ __( 'The position at which to insert the prompt.', 'newspack-popups' ) }
```